weave

```diff
@@ -1382,13 +1382,15 @@ check_overlap() {
 # Claim addresses for a container in IPAM.  Expects to be called from
 # with_container_addresses.
 ipam_reclaim() {
-    for CIDR in $3 ; do
-        http_call $HTTP_ADDR PUT "/ip/$1/$CIDR?noErrorOnUnknown=true&?check-alive=true" || true
+    CONTAINER_ID="$1"
+    [ "$2" = "$CONTAINER_IFNAME" ] || CONTAINER_ID="_"
```

Comment:
I have inverted this from `!=` and `&&`; I generally prefer using a disjunction in cases like this since it ensures that the exit status is 0.